proof/refine/AARCH64/InvariantUpdates_H.thy

```diff
@@ -56,8 +57,9 @@ lemma invs_no_cicd'_machine:
 proof -
   show ?thesis
     apply (cases "ksSchedulerAction s")
-    apply (simp_all add: all_invs_but_ct_idle_or_in_cur_domain'_def valid_state'_def cur_tcb'_def ct_in_state'_def ct_idle_or_in_cur_domain'_def tcb_in_cur_domain'_def
-                    valid_queues_def valid_queues_no_bitmap_def bitmapQ_defs
+    apply (simp_all add: all_invs_but_ct_idle_or_in_cur_domain'_def valid_state'_def
+                    cur_tcb'_def ct_in_state'_def ct_idle_or_in_cur_domain'_def tcb_in_cur_domain'_def
```

Comment:
the `cur_tcb'_def` doesn't follow indent guidelines, should be right of `:` (it already didn't, but since we're changing the start of the command, might be worth a fix)